scripts/ci-entrypoint.sh

```diff
@@ -58,7 +58,7 @@ setup() {
         echo "Will use the ${IMAGE_REGISTRY}/${CCM_IMAGE_NAME}:${IMAGE_TAG} cloud-controller-manager image for external cloud-provider-cluster"
         echo "Will use the ${IMAGE_REGISTRY}/${CNM_IMAGE_NAME}:${IMAGE_TAG} cloud-node-manager image for external cloud-provider-azure cluster"
 
-        export CCM_IMG_ARGS=(--set cloudControllerManager.imageRepository="${IMAGE_REGISTRY}"
+        CCM_IMG_ARGS=(--set cloudControllerManager.imageRepository="${IMAGE_REGISTRY}"
```

Comment:
verified this fixes the issue on macOS @jackfrancis

Reply:
This is a very simple fix, thanks!

Is the root misunderstanding here that you must export a variable in order to re-use it in another function scope? I see it is declared and assigned a value in the `setup` func, and then referenced in the `install_cloud_provider_azure` func.

Reply:
Right, here this variable doesn't need to be exported since it's only used later in the same script. In most cases the `export` would be harmless but in this case because arrays + export doesn't work with older versions of bash we're better off not exporting.

Reply:
Nothing makes me feel better than being reminded that our world depends on code that uses variables with no function scope.